From ee93dde8cfc53e41a5bb04f88a5bb64563247569 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:40:01 -0400
Subject: [PATCH 11/13] makechrootpkg: don't delete PACKAGER and fix MAKEFLAGS

Keep MAKEFLAGS so it can prefix a call to makechrootpkg.
MAKEFLAGS= must not be defined in $BUILDROOT/etc/makepkg.conf for this to work.
See: https://bugs.archlinux.org/task/64938
---
 src/makechrootpkg.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/makechrootpkg.in b/src/makechrootpkg.in
index b906c82..aadb17a 100644
--- a/src/makechrootpkg.in
+++ b/src/makechrootpkg.in
@@ -176,9 +176,8 @@ prepare_chroot() {
 
 	$install -d "$copydir"/{build,startdir,{pkg,srcpkg,src,log}dest}
 
-  sed -e '/^MAKEFLAGS=/d' -e '/^PACKAGER=/d' -i "$copydir/etc/makepkg.conf"
   for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest \
-    "MAKEFLAGS='${MAKEFLAGS:-}'" "PACKAGER='${PACKAGER:-}'"
+    "MAKEFLAGS='${MAKEFLAGS:-}'"
 	do
 		grep -q "^$x" "$copydir/etc/makepkg.conf" && continue
 		echo "$x" >>"$copydir/etc/makepkg.conf"
-- 
2.45.0

